website/app/styles/pages/application/header.scss

```diff
@@ -163,46 +163,29 @@
   margin: 0 0 0 auto;
 }
 
-// "fake" search button (this triggers another button, the "Algolia search" one, which is hidden)
-
-.doc-page-header__search-button {
+.doc-page-header__desktop-icon-item {
   @include doc-font-style-navigation();
   @include doc-header-custom-focus ();
   display: flex;
```

Comment:
[question] does this still need to be a flex element?

Reply:
ah, it doesn't! I will remove.